index.js

```diff
@@ -32,8 +36,8 @@ function checkAndGetDimensionValue(value, percentageBase) {
 }
 
 function calculateWidthHeight(imageWidth, imageHeight, inputWidth, inputHeight, preserveAspectRatio) {
-	const terminalColumns = process.stdout.columns || 80;
-	const terminalRows = process.stdout.rows || 24;
+	const terminalColumns = process.stdout.columns - 2 || 80;
+	const terminalRows = process.stdout.rows - 2 || 24;
```

Comment:
Why?

Reply:
Images always seem to be rendered one character larger than terminal height. log-update automatically tries to strip the top layer so that the text can fit in the `fitToTerminalHeight` function. The problem with this is that the automatic trimming doesn't seem to work and causes the entire frame to not render.

Reply:
Would be great if you could open an issue on `log-update` about that if it's actually an issue, and link to the issue as a comment in the code here.

Reply:
I'm not quite sure where the fundamental issue lies. To reproduce, change the lines so that they don't subtract 2 and run in a non-iTerm environment.

Reply:
Would be great if you could look into it and open an issue. I don't like merging workarounds that will never be properly fixed.